docs/_articles/en/v3.x/salesforce-graph-engine/rules.md

```diff
@@ -92,11 +128,11 @@ Graph Engine encountered an error while walking this path. Manually verify that
 - [Filter SOQL Queries Using WITH SECURITY_ENFORCED](https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_classes_with_security_enforced.htm)
 - [Frequently Asked Questions](./en/v3.x/faq/)
 
-## ApexNullPointerExceptionRule
+### ApexNullPointerExceptionRule
```

Comment:
Again, you'll need an anchor tag here to preserve our links.
`### ApexNullPointerExceptionRule <a name='ApexNullPointerExceptionRule'>#</a>`

Reply:
@jfeingold35 I just added custom anchor tags throughout the whole doc